File: app/src/main/java/me/rytek/liveshoppinglist/Item.java
package me.rytek.liveshoppinglist;

public class Item {
    public String id;
    public String item;
    public int quantity;
    public Boolean bought;
    public String added_by;
    public String comments;

    public Item(String item, int quantity, Boolean bought, String added_by, String comments) {
        this.item = item;
        this.quantity = quantity;
        this.bought = bought;
        this.added_by = added_by;
        this.comments = comments;
    }
}

File: app/src/main/java/me/rytek/shoppinglist/model/Item.kt
package me.rytek.shoppinglist.model


import com.squareup.moshi.JsonClass
import java.util.*

@JsonClass(generateAdapter = true)
data class Item (
    var id: Long,
    var name: String,
    var quantity: Long,
    var comments: String,
    var markedOff: Boolean,
    var isDeleted: Boolean,
    var timeAdded: Date,
    var timeMarkedOff: Date?,
    var timeDeleted: Date?,
    var addedBy: User
)